src/EditorFeatures/CSharpTest/Wrapping/AbstractWrappingTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Wrapping
+{
+    public abstract class AbstractWrappingTests : AbstractCSharpCodeActionTest
+    {
+        protected sealed override ImmutableArray<CodeAction> MassageActions(ImmutableArray<CodeAction> actions)
+            => FlattenActions(actions);
+
+        protected static Dictionary<OptionKey, object> GetIndentionColumn(int column)
+            => new Dictionary<OptionKey, object>
+               {
+                   { FormattingOptions.PreferredWrappingColumn, column }
+               };
+
+        protected Task TestAllWrappingCasesAsync(
+            string input,
+            params string[] outputs)
+        {
+            return TestAllWrappingCasesAsync(input, options: null, outputs);
+        }
```

Comment:
wrapping fixes are interrelated.  For example, we won't offer one fix if it would end up with the same net effect as a previously added one.  As such, the tests actually verify the entire set of expected refactorings offered.  This verifies

1. we're not getting less/more than we want
2. we're getting things in the right order
3. we're not getting dupes.

It also helps as testing things by manually passing indices is quite confusing, especially as the list is so dynamic.